screensaver.turnoff/addon.xml

```diff
@@ -1,13 +1,49 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.turnoff" name="Turn Off" version="0.9.1" provider-name="Dag Wieers">
+<addon id="screensaver.turnoff" name="Turn Off" version="0.10.2" provider-name="dagwieers">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.ui.screensaver" library="default.py"/>
   <extension point="xbmc.addon.metadata">
     <platform>all</platform>
-    <summary lang="en_GB">Screensaver that turns your screen off to save power</summary>
-    <description lang="en_GB">The Turn Off screensaver turns your TV, projector or monitor off, like any old fashioned screensaver is intended to do.</description>
-    <source>http://github.com/dagwieers/plugin.screensaver.off</source>
-    <license>GPL2+</license>
+    <summary lang="en_GB">Screensaver that turns your display off to save power</summary>
+    <description lang="en_GB">
+The Turn Off screensaver turns your TV, projector or screen off, like any old fashioned screensaver is intended to do.
+
+Next to managing your display, it can also manage your device power state, log your profile off or mute audio to avoid sounds through your A/V receiver.
+    </description>
+    <license>GNU General Public License, v2+</license>
```

Comment:
Please use spdx

Reply:
I will fix this in the next release. Thanks.

*It doesn't seem to be a requirement, as it is missing from the docs and just recently other add-ons were merged that did not use SPDX for their license-tag.
https://github.com/xbmc/addon-check/issues/126*

Reply:
I documented on the wiki some months ago: https://kodi.wiki/view/Addon.xml#.3Clicense.3E
 It's not mandatory but it is nice to start having some sort of standardization. Might be added to the addon checker if someone steps up.
Thanks for adjusting

Reply:
Oh, I agree with having SPDX as a standard.